bpf/lib/common.h

```diff
@@ -328,6 +328,9 @@ enum {
 /* Cilium error codes, must NOT overlap with TC return codes.
  * These also serve as drop reasons for metrics,
  * where reason > 0 corresponds to -(DROP_*)
+ *
+ * These are shared with pkg/monitor/api/drop.go, when modifying any of the
+ * below, that file should also be updated.
```

Comment:
On the way, could you add a comment in _pkg/monitor/api/drop.go_ `errors` that DROP values are also shared in _api/v1/flow/flow.proto_ `enum DropReason` please?